### PR TITLE
Requested changes for publishing in the Registry

### PR DESCRIPTION
**Pull request by ringods**
Hello @pandatix,

Here are a few changes we would like to see integrated in your provider before we can go forward with publishing it in the Pulumi Registry. 

Together with the code changes, we would like to indicate that some of the name or namespace for some of the generated SDK packages needs to change. You changed it for your JS/TS package and would like to see the same for your Python and .NET Nuget package. At Pulumi, we like to reserve the package name or prefix `pulumi` for ourselves.

Have a look in this section how a community effort documented their naming scheme for all the language SDKs:
https://github.com/pulumiverse/infra/blob/main/README.md#package-registries

After reviewing these, let me know if you have any further questions.

Ringo

### CODE REVIEW

File: docs/installation-configuration.md [hunk 1]
```diff
@@ -37,7 +47,7 @@ export CTFD_API_KEY="ctfd_xxx"
 We recommend using an API key rather than a nonce/session combo for security purposes: the API key is natively supported thus enable better logging, authentication & authorization.
 Moreover, it is possible to rotate the keys and revoke them on the fly using the API, while a session/nonce is not build in this way.
 
-### Configuring
+### Configuring
```
Comment (pandatix) [inline docs/installation-configuration.md:50]: Nice catch !

**Review by pandatix — APPROVED** [review]
LGTM

**Comment by pandatix** [comment]
Hi, thanks for your feedbacks and comments, I have to admit I based this effort on the muhlba91 provider for proxmoxve so we were prone to technical debt despite all the amazing work he does every day !

**Comment by ringods** [comment]
@pandatix at our side, we also upped the requirements to get listed, hence it might deviate from some of the existing published packages. But I will work with you to get this succesfully published.

**Pull request merged**